test/legacy/roles/scaleway_compute/tasks/main.yml

```diff
@@ -0,0 +1,388 @@
+# SCW_API_KEY='XXX' SCW_ORG='YYY' ansible-playbook ./test/legacy/scaleway.yml --tags test_scaleway_compute
+
+- name: Create a server (Check)
+  check_mode: yes
+  scaleway_compute:
+    name: foobar
+    state: present
+    image: 89ee4018-f8c3-4dc4-a6b5-bca14f985ebe
```

Comment:
For the sake of maintainability, I think this should be in a variable, and some comment on where to get it, if the uuid is changed later.

Reply:
I would add the same for region anc commercial_type

Reply:
done